### DISCUSSION TITLE
Using Filament forms with Volt Guide help

### DISCUSSION THREAD

**Discussion by GorchPortley** [thedevdojo/wave#167] [General]
Hello! I am working on integrating filament forms and tables on a single page in my users dashboard like in the , when i look at the reference code for the view portion of the volt page, it seems to use <x-modal which doesn't seem to exist, is this a custom component not included in the wave source?

https://devdojo.com/wave/docs/guides/using-filament-with-volt#combining-the-table-and-form-builder

**Comment by eloquentbit** [comment]
I have the same issue:
`Unable to locate a class or view for component [modal].`

OS: macOs Sonoma 14.7
Php: 8.4.1
Laravel: 11.31.0

**Comment by eloquentbit** [comment]
@GorchPortley  I have found the solution: the correct tag for the modal is `<x-filament::modal>` and not `<x-modal> `

**Reply by bobbyiliev** [reply]
Thank you @eloquentbit for sharing your solution here! Very much appreciated!
